prov/efa/src/efa_prov.c

```diff
@@ -184,13 +184,18 @@ EFA_INI
 	if (g_device_cnt <= 0)
 		return NULL;
 
+	/*
+	 * efa_env_initialize uses g_efa_device_list
+	 * so it must be called after efa_device_list_initialize()
+	 */
+	efa_env_initialize();
```

Comment:
Is it possible that an error could occur and needs handling?

Reply:
No, there won't be error from this function.

Reply:
It mainly call fi_param_get which either assign the enviroment value to efa_env or silently ignore if the the environment variable is misconfigured.

Reply:
OK, cool, thanks for checking